src/views/studio/studio.scss

```diff
@@ -0,0 +1,98 @@
+@import "../../colors";
+@import "../../frameless";
+
+$radius: 8px;
+
+.studio-page {
+    background-color: #E9F1FC;
```

Comment:
Is there a color variable in `_colors.scss` we could use instead?

Reply:
`$ui-blue-10percent` on top of `$background-color` creates the same color.